src/main/ipc/quick-css.ts

```diff
@@ -6,7 +6,9 @@ import { CONFIG_PATHS } from "src/util";
 
 const CSS_PATH = join(CONFIG_PATHS.quickcss, "main.css");
 
-ipcMain.handle(RepluggedIpcChannels.GET_QUICK_CSS, () => readFile(CSS_PATH, { encoding: "utf-8" }));
+ipcMain.handle(RepluggedIpcChannels.GET_QUICK_CSS, () =>
+  readFile(CSS_PATH, { encoding: "utf-8" }).catch(() => ""),
```

Comment:
Shouldn't there be better errorhandling? Maybe make the file if it's not found?

Reply:
Already added that.

Reply:
That's on discord load... not here?

Reply:
Discord load happens before this. Adding it here will make no difference.